app/controllers/miq_policy_controller/alert_profiles.rb

```diff
@@ -110,9 +130,8 @@ def alert_profile_assign
   def alert_profile_delete
     alert_profiles = []
     # showing 1 alert set, delete it
-    if params[:id].nil? || MiqAlertSet.find_by_id(params[:id]).nil?
-      add_flash(_("%{model} no longer exists") % {:model => ui_lookup(:model => "MiqAlertSet")},
-                :error)
+    if params[:id].nil? || !MiqAlertSet.exist?(params[:id])
```

Comment:
Change it to `exists?`, please. `exist?` does no exist :grimacing:

Reply:
thx!